homeassistant/components/statsd.py

```diff
@@ -57,8 +57,16 @@ def statsd_event_listener(event):
         except ValueError:
             return
 
+        states = dict(state.attributes)
+
         _LOGGER.debug('Sending %s.%s', state.entity_id, _state)
-        statsd_client.gauge(state.entity_id, _state, sample_rate)
+        statsd_client.gauge("%s.state" % state.entity_id, _state, sample_rate)
```

Comment:
I am not against breaking changes if we can't avoid it but always urge to explore avoiding it.